Git-commit: c34d1b22fef329d5cecd003d7be249937ec70a3c
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Fri, 28 Oct 2022 13:02:36 +0200
Subject: [PATCH] Linux 5.10.151
References: bsn#19
Patch-mainline: v5.10.151

Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 0ecf8333cc17..0e22d4c8bc79 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 150
+SUBLEVEL = 151
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

