Git-commit: 37050f17f2d243f0d6ebe63ed3df84f91abce614
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Thu, 16 Dec 2021 16:39:01 +0100
Subject: [PATCH] Linux 5.10.86
References: bsn#19
Patch-mainline: v5.10.86

Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 8de80228df3f..5c1a33f1ecad 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 85
+SUBLEVEL = 86
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.34.1

